{
  "0 debug pnpm:scope": {
    "selected": 1
  },
  "1 info pnpm": {
    "err": {
      "name": "Error",
      "message": "not found: dev",
      "code": "ENOENT",
      "stack": "Error: not found: dev\n    at getNotFoundError (D:\\nodejs\\node_global\\node_modules\\pnpm\\dist\\pnpm.cjs:41517:51)\n    at Function.whichSync [as sync] (D:\\nodejs\\node_global\\node_modules\\pnpm\\dist\\pnpm.cjs:41594:13)\n    at getCommandAbsolutePathSync (D:\\nodejs\\node_global\\node_modules\\pnpm\\dist\\pnpm.cjs:43159:44)\n    at default_1 (D:\\nodejs\\node_global\\node_modules\\pnpm\\dist\\pnpm.cjs:43168:32)\n    at D:\\nodejs\\node_global\\node_modules\\pnpm\\dist\\pnpm.cjs:171981:39"
    }
  },
  "2 error pnpm": {
    "code": "ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL",
    "prefix": "C:\\Users\\zxb",
    "err": {
      "name": "pnpm",
      "message": "not found: dev",
      "code": "ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL",
      "stack": "Error: not found: dev\n    at getNotFoundError (D:\\nodejs\\node_global\\node_modules\\pnpm\\dist\\pnpm.cjs:41517:51)\n    at Function.whichSync [as sync] (D:\\nodejs\\node_global\\node_modules\\pnpm\\dist\\pnpm.cjs:41594:13)\n    at getCommandAbsolutePathSync (D:\\nodejs\\node_global\\node_modules\\pnpm\\dist\\pnpm.cjs:43159:44)\n    at default_1 (D:\\nodejs\\node_global\\node_modules\\pnpm\\dist\\pnpm.cjs:43168:32)\n    at D:\\nodejs\\node_global\\node_modules\\pnpm\\dist\\pnpm.cjs:171981:39"
    }
  }
}